Source/Windows/ARM64EC/Module.cpp

```diff
@@ -45,21 +46,29 @@ desc: Implements the ARM64EC BT module API using FEXCore
 #include <wine/debug.h>
 
 class ECSyscallHandler;
+
+extern "C" {
 void* X64ReturnInstr; // See Module.S
 extern void* ExitFunctionEC;
 
+// Wine doesn't support issuing direct system calls with SVC, and unlike Windows it doesn't have a 'stable' syscall number for NtContinue
+void* WineSyscallDispatcher;
```

Comment:
You probably want to `alignas(4096)` on this so that `adrp` in the Module.S is always guaranteed to have correct alignment when loading this address.

Reply:
Er wait, misread, you do lo12 offset. nevermind.